packages/telemetry/src/serialize-slog-obj.js

```diff
@@ -1,4 +1,26 @@
-export const serializeSlogObj = slogObj =>
-  JSON.stringify(slogObj, (_, arg) =>
-    typeof arg === 'bigint' ? Number(arg) : arg,
-  );
+const { hasOwn } = Object;
+
+const replacer = (_key, value) => {
+  switch (typeof value) {
+    case 'object': {
+      if (value instanceof Error) {
```

Comment:
A motivating comment may be useful here:

```suggestion
      if (value instanceof Error) {
        // Recursively expand Errors so that ...
```

Reply:
+1. Also it took me a minute to remember that a recursive call to `replacer` was necessary.

Reply:
Added the comment, and also updated the replacement object to `{ errorType, message, cause?, errors?, stack? }` for better readability by a possibly-just-awoken engineer.